README.md

```diff
@@ -16,16 +16,16 @@ glide_version := v0.13.1<br/>
 glide_arch := linux-amd64  (change that to your architecture depends on the OS)
```

Comment:
this is makefile syntax and it's not explained anywhere, people will be confused. if you want this whole section to be clearer transform it to bash:

```bash
$ export GLIDE_VERSION = "v0.13.1"
# set architecture depending on your machine
# set architectures in https://github.com/Masterminds/glide/releases
$ export GLIDE_ARCH = "linux-amd64"

$ curl -sSLo glide.tar.gz https://github.com/Masterminds/glide/releases/download/$(GLIDE_VERSION)/glide-$(GLIDE_VERSION)-$(GLIDE_ARCH).tar.gz
$ tar -xf ./glide.tar.gz
$ mv ./$(GLIDE_ARCH)/glide ./glide
$ rm -rf ./$(GLIDE_ARCH) ./glide.tar.gz
$ glide install
```